canvas_account_admin_tools/settings/local.py

```diff
@@ -99,6 +99,26 @@
    'icommons_rest_api': {
       'base_path': 'api/course/v2'
    },
+
+    'publish_courses': {
+        'test_course': {
+            'relative_url': 'courses/2601'
+        },
+        'test_terms': {
+            'term_with_unpublished_courses': 'Summer 2017',
```

Comment:
suggestion for brevity -- these are subkeys of `test_terms`, so the `term_` bit of this key could probably be dropped without losing readability in the code.